### PR TITLE
Re-add the Disable TLS certificate verification tutorial

### PR DESCRIPTION
**Pull request by majakurcius**
**Description**

As the bug with disabling TLS certificate verification has been fixed, the tutorial instructing the user how to do it should be re-added. 

Changes proposed in this pull request:

- Re-add the "Disable TLS certificate verification" tutorial
- Add the **spec.skipVerify** field to the Application CR documentation

**Related issue(s)**
#11946

### CODE REVIEW

File: docs/03-tutorials/00-application-connectivity/ac-11-disable-tls-certificate-verification.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Disable TLS certificate verification
+---
+
+You can disable the [TLS certificate verification](../../01-overview/main-areas/application-connectivity/ac-04-security.md#tls-certificate-verification) for the connections between Kyma and external solution represented by an application. This allows Kyma to send requests and data to an unsecured application without verifying its presented TLS certificate. Disabling the certificate verification can be useful in certain testing scenarios.
```
Comment (NHingerl) [inline docs/03-tutorials/00-application-connectivity/ac-11-disable-tls-certificate-verification.md:5]:
```suggestion
You can disable the [TLS certificate verification](../../01-overview/main-areas/application-connectivity/ac-04-security.md#tls-certificate-verification) for the connections between Kyma and an external solution represented by an application. This allows Kyma to send requests and data to an unsecured application without verifying its presented TLS certificate. Disabling the certificate verification can be useful in certain testing scenarios.
```

**Review by klaudiagrz — DISMISSED** [review]

File: docs/03-tutorials/00-application-connectivity/ac-11-disable-tls-certificate-verification.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+You can disable the [TLS certificate verification](../../01-overview/main-areas/application-connectivity/ac-04-security.md#tls-certificate-verification) for the connections between Kyma and an external solution represented by an Application. This allows Kyma to send requests and data to an unsecured Application without verifying its presented TLS certificate. Disabling the certificate verification can be useful in certain testing scenarios.
+
+>**NOTE:** By default, the TLS certificate verification is enabled when sending data and requests to every Application.
+Follow these steps to disable TLS certificate verification:
+
+1. Edit the `{APPLICATION_CR_NAME}` Application CR. Run:
+
+   ```bash
+   kubectl -n kyma-integration edit application {APPLICATION_CR_NAME}
```
Comment (franpog859) [inline docs/03-tutorials/00-application-connectivity/ac-11-disable-tls-certificate-verification.md:13]:
```suggestion
   kubectl edit application {APPLICATION_CR_NAME}
```

**Review by klaudiagrz — APPROVED** [review]

**Review by franpog859 — DISMISSED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Comment by majakurcius** [comment]
image

**Pull request merged**